h2/src/test/org/h2/test/scripts/TestScript.java

```diff
@@ -73,6 +75,10 @@ public static void main(String... a) throws Exception {
 
     @Override
     public void test() throws Exception {
+        Path currentRelativePath = Paths.get("");
+        String currentAbsolutePath = currentRelativePath.toAbsolutePath().toString();
+        System.out.println("Current absolute path is: " + currentAbsolutePath);
+        
```

Comment:
debugging leftovers?

Reply:
This test class, annoyingly, wont run outside the TestAll scenario in Eclipse. I want to test my tests standalone outside of TestAll. Whats the trick ? I couldn't get it to work...